recursion/towersOfHanoi/Solution1Recursion.go

```diff
@@ -0,0 +1,16 @@
+package TowersOfHanoi
+
+import "fmt"
+
+type ImplS1R struct{}
+
+var out string
```

Comment:
any reason you are declaring out as a global variable? I do not see any; instead just return the string from the method